tools/helm/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG base_image
+
+FROM ${base_image}
+
+ENV helm_version 3.7.1
+
+RUN clean-install jq curl ca-certificates
+
+RUN curl https://get.helm.sh/helm-v${helm_version}-linux-amd64.tar.gz -o helm.tar.gz && \
```

Comment:
does osm plan on supporting multi-arch like arm64? if so, we should also make platform configurable